ci/testkomodo.sh

```diff
@@ -0,0 +1,14 @@
+
+copy_test_files () {
+    cp -r $CI_SOURCE_ROOT/tests $CI_TEST_ROOT/tests
+    cp $CI_SOURCE_ROOT/pyproject.toml $CI_TEST_ROOT
+}
+
+install_test_dependencies () {
+    pip install -e ".[ert]"
+    pip install pytest
```

Comment:
Komodo (quite recently) added pytest by default, so this single line is not needed. Unless  any special pytest packages  are needed to run tests, then this bash function can be empty.

Unless this function is defined, one gets the default function implementation at:

https://github.com/equinor/komodo/blob/b6be86ded198956312e8304a4729ef628355997b/ci/func_def.sh#L19-L25

and is then probably all you need then for completor.